docs/sources/googlefirestore.md

```diff
@@ -2,6 +2,10 @@
 
 For building ContainerSource for Google Cloud Firestore that pulls events from Firestore and sends them to Knative service/channel
 
+## Prerequisite(s)
+
+- Google Cloud Firestore
+
```

Comment:
Not part of our public facing docs, can be removed.

Reply:
@odacremolbap are you meaning to delete the whole file?

Reply:
yes delete